feature_updates/src/main/java/cm/aptoide/pt/feature_updates/domain/Updates.kt

```diff
@@ -109,4 +117,23 @@ class Updates @Inject constructor(
 
   private fun getInstalledApps(): List<PackageInfo> =
     packageManager.getInstalledPackages().filter { it.ifNormalAppOrGame() }
+
+  suspend fun autoUpdate() {
+    getAppsUpdates()
+      .first()
+      .filter {
+        if (VERSION.SDK_INT >= VERSION_CODES.UPSIDE_DOWN_CAKE) {
+          packageManager.getInstallSourceInfo(it.packageName).updateOwnerPackageName == applicationContext.packageName
+        } else if (VERSION.SDK_INT >= VERSION_CODES.R) {
+          packageManager.getInstallSourceInfo(it.packageName).installingPackageName == applicationContext.packageName
+        } else {
+          false
+        } || it.packageName == applicationContext.packageName
+      }
+      .forEach {
+        installManager.getApp(it.packageName).install(
```

Comment:
This should use constraints for Wifi only downloads

Reply:
Fixed